src/pybind/mgr/dashboard_v2/controllers/rgw.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+
+from ..tools import ApiController, RESTController
+from .. import logger
+
+
+@ApiController('rgw')
+class Rgw(RESTController):
```

Comment:
I think this API should require an authentication, so please add ``@AuthRequired()``